flux-tests/tests/todo/struct_types/I143.rs

```diff
@@ -9,8 +9,12 @@ pub struct Pair {
     pub y: i32,
 }
 
-#[flux::sig(fn(Pair[@x]) -> i32)]
-pub fn mytuple(p:Pair) -> i32 {
+#[flux::sig(fn(Pair[@p,@q,@r]) -> i32[p])]
```

Comment:
This file has already been considered in other tests so we can remove it, no?

Reply:
Yes let me remove.